user-guide/Troubleshooting/Procedures/Cassandra_expired_blocking_SSTables.md

```diff
@@ -0,0 +1,63 @@
+---
+uid: Cassandra_expired_blocking_SSTables
+---
+
+# Expired SSTables blocking other tables from being dropped
+
+In case the only data SSTables in Cassandra contain is tombstones, and the tables are therefore expired, they could block other tables that are also expired from being dropped. This could result in various issues, such as an increase in the disk usage of the Cassandra data partition, or memory leak behavior in the TWCS keyspaces (*trend_data_medium*, *trend_data_short*, and *trend_data_long*).
+
+You can check if you are indeed encountering this issue using the Cassandra tools, for example with the following commands (on Linux):
```

Comment:
Please add that that the following commands will stop the Cassandra service and therefore could impact the working of any system using the DB (e.g. DataMiner).